packages/dara-core/dara/core/internal/websocket.py

```diff
@@ -195,7 +211,7 @@ async def process_client_message(self, message: ClientMessage):
                 eng_logger.error(f'No handler found for custom message kind {kind}', e)
                 return
 
-    async def send_and_wait(self, message: ServerMessage) -> Optional[ClientMessage]:
+    async def send_and_wait(self, message: ServerMessage) -> Optional[Any]:
```

Comment:
What's the reason to use Any here? I thought ClientMessage was pretty loosely-defined anyway

Reply:
This is because we don't actually return the ClientMessage object, we return ClientMessage.message which has an Any type. This is just matching the type to what we actually return. Is there a way to do `ClientMessage.message` as the type? I tried it that way and as `ClientMessage['message']` but both broke in different ways...

Reply:
Ah I see. Yeah there isn't a way to do that in Python, I don't think 😅 Happy to keep as is